gatorgrade/output/output_functions.py

```diff
@@ -98,3 +98,10 @@ def output_failed_checks(failed_checks):
         print(f"{color.Fore.RED}\u2718  {color.Style.RESET_ALL}{requirement}")
         print(f"    {color.Fore.YELLOW}\u2192  {description}")
         return bool
+
+
+def run_and_display_command_checks(commands):
+    """Run commands through gatorgrader and display them to the user."""

```

Comment:
Could the docstring include an `Args` section that describes what the value of `commands` is expected to be?

Reply:
Would it make more sense for me to put the `Args` section from the `run_commands_and_return_results()` function down in this spot instead of where it is?  Since the argument being passed in is the same for both functions I don't want to run the risk of repeating myself unless you think that's alright.

Reply:
@DlHolland, I think that it would be fine to repeat the comment in both places so that someone looking at the code will understand each function in isolation. What do you think?

Reply:
That works for me!  Just wanted to make sure it wouldn't be an issue.